app/views/fileChecksProgress.scala.html

```diff
@@ -9,8 +9,14 @@
             @progressIndicator(Messages("checkingRecords.progress"))
             <h1 class="govuk-heading-xl">@Messages("checkingRecords.title")</h1>
             <p class="govuk-body">@Messages("checkingRecords.description")</p>
-            <p class="govuk-body">@Messages("checkingRecords.antivirus")</p>
-            <progress id="av-metadata-progress-bar" class="progress-display" value="@avMetadataPercentage" max="100"></progress>
+            <p>
```

Comment:
Could the `<progress>` element be removed from the `<p>` element? Normally, paragraph elements should just contain text - it will make any custom CSS less confusing if we keep the text and other visual displays separate.

Reply:
Ah, yes, I added the <p> tag around them because without them, the separation between the two bars looked a bit off. Maybe I should play around with the <br> tag or line-height instead?

Reply:
Would you like to pair on this later today? I've got a few ideas for how to approach it, and it would be good to work it out together.

Reply:
Sure!